js/index.js

```diff
@@ -0,0 +1,85 @@
+const menuButton = document.querySelector('.nav-ham-icon');
+const bar = document.getElementsByClassName('bar');
+const navSocial = document.querySelector('.nav-social');
+const navBar = document.querySelector('.nav-bar');
+const header = document.querySelector('.nav');
+const headerItems = document.getElementsByClassName('toggle1');
+const speaker = document.querySelector('.speaker-container');
+
+menuButton.addEventListener('click', () => {
+  Array.from(bar).forEach((element) => {
+    element.classList.toggle('pressed');
+  });
+  navSocial.classList.toggle('active');
+  navBar.classList.toggle('active');
+  menuButton.classList.toggle('pressed');
+  header.classList.toggle('pressed');
+});
+
+Array.from(headerItems).forEach((element) => {
+  element.addEventListener('click', () => {
+    if (bar[0].classList.contains('pressed')) {
+      Array.from(bar).forEach((element) => {
+        element.classList.toggle('pressed');
+      });
+      navSocial.classList.toggle('active');
+      navBar.classList.toggle('active');
+      menuButton.classList.toggle('pressed');
+      header.classList.toggle('pressed');
+    }
+  });
+});
+
+const speakerList = [
+  {
+    img: 'Images/s1.png',
+    name: 'Rajendra Singh',
+    des1: 'Indian water conservationist and environmentalist.',
+    des2: 'Also known as "waterman of India", he won the Magsaysay Award in 2001.',
+  },
+  {
+    img: 'Images/s1.png',
+    name: 'Amla Ruia',
+    des1: 'Amla Ruia, a Mumbai-based social activist',
+    des2: 'she founded Aakar Charitable Trust. Up until now, the Trust has helped in the construction of 2000',
+  },
+  {
+    img: 'Images/s1.png',
+    name: 'Aabid Surti',
+    des1: 'Aabid Surti runs a one-man NGO named the Drop Dead Foundation',
+    des2: 'The 80-year-old does it all for free, with his team of one volunteer and a plumber.',
+  },
+  {
+    img: 'Images/s1.png',
+    name: 'Ayyappa Masagi',
+    des1: 'Ayyappa Masagi has changed the lives of thousands',
+    des2: 'He started planting rubber and coffee',
+  },
+  {
+    img: 'Images/s1.png',
+    name: 'Shirish Apte',
+    des1: 'An executive engineer with the Minor Irrigation',
+    des2: 'Recharged groundwater levels and increased agricultural output',
+  },
+  {
+    img: 'Images/s1.png',
+    name: 'Chandra Kishore Patil',
+    des1: 'Chandra Kishore Patil, a resident of Indiranagar, Nashik',
+    des2: 'Despite him being faced by rude people',
+  },
+
+];
```

Comment:
- [ ] [Optional] I would suggest you please implement a different image for the `feature speaker` as it will help other people to identify the person.
### Design
![Screenshot 2022-11-25 224326](https://user-images.githubusercontent.com/81727988/204030070-3e709f4f-0f56-465c-a03c-ed28f6900798.png)
### Design provided by Microverse
![Figma](https://user-images.githubusercontent.com/81727988/204030126-fb933045-77ac-4750-ba44-26524b4d1c68.png)